custom-connectors/Assently E-Sign/Intro.md

```diff
@@ -0,0 +1,16 @@
+﻿
```

Comment:
@kian-beetroot Please delete intro.md as it is not required for open sourcing your connector.

Reply:
deleted as requested